src/Enghouse/client/indexWatson.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
```

Comment:
I think I would suggest renaming this to `watsonExample.html` to make it a little more clear that this is an example of an html file that uses your integration and not necessarily a file a customer has to use directly.

Reply:
This has now been done